lib/cli/utils.js

```diff
@@ -88,7 +88,8 @@ utils.iterativeCompletion = (inputs, depth, completer) => {
   // they'll be not valid (those are completions for previous inputs)
   if (newDepth <= inputs.length - 1) completions = [];
   if (!completions[0] ||
-      completions.length === 1 && completions[0] === inputs[depth]) {
+    completions.length === 1 && completions[0] === inputs[depth]
```

Comment:
@tshemsedinov I saw that, but the code in the first part of your message is not how @belochub and me wanted to align it, so I elaborated to make sure that you've seen the [correct version](https://github.com/metarhia/jstp/pull/244#discussion_r126941449).